controllers/search_controller.rb

```
@@ -13,7 +13,40 @@ class SearchController < ApplicationController
+      resp = search(Ontology, query, options)
+
+      result = {}
+      acronyms_ids = {}
+      resp.each do |doc|
+        resource_id = doc["resource_id"]
+        id = doc["submissionId_i"]
+        acronym = doc["ontology_t"].split('/').last
+        next if acronym.blank?
+        
+        already_found_submission_id = acronyms_ids[acronym]
+        old_id = already_found_submission_id.to_i rescue 0
```

RuboCop (1.73)

[convention] Avoid using `rescue` in its modifier form.

(Style/RescueModifier)